### PR TITLE
Add density strings for all current DPIs

### PR DESCRIPTION
**Pull request by NightlyNexus**
Oops, looks like I had this sitting for a year in a feature branch that I didn't finish.

### CODE REVIEW

File: telescope/src/main/java/com/mattprecious/telescope/EmailDeviceInfoLens.java [hunk 1]
```diff
@@ -107,12 +107,24 @@ public static String getDensityString(DisplayMetrics displayMetrics) {
         return "ldpi";
       case DisplayMetrics.DENSITY_MEDIUM:
         return "mdpi";
+      case DisplayMetrics.DENSITY_TV:
```
Comment (mattprecious) [inline telescope/src/main/java/com/mattprecious/telescope/EmailDeviceInfoLens.java]: This is handled below.

Comment (NightlyNexus) [inline telescope/src/main/java/com/mattprecious/telescope/EmailDeviceInfoLens.java]: Updated. Moved up to stay in order of density; should I move it down?

**Comment by NightlyNexus** [comment]
25 also added more buckets. I could send a PR for upping the compileSdkVersion and such, if you'd like. (Sorry to ping the old thread.)

**Comment by mattprecious** [comment]
Sure! I'll try to be better about merging it in a timely manner if you do 😄

**Comment by mattprecious** [comment]
(Thanks!)

**Pull request merged**